models/checklist.js

```diff
@@ -4,7 +4,7 @@ const Schema = mongoose.Schema
 
 const checklistSchema = new mongoose.Schema({
     owner: {type: Schema.Types.ObjectId, ref: "Profile"},
-    name: {type: String},
+    checklistName: {type: String},
```

Comment:
@vzarate-94 Why change `name` to `checklistName`? Since we are in the `Checklist` model, `name` is probably easier to reference.